README.md

```diff
@@ -30,9 +30,86 @@ If `var/chaindata` does not exist, the `default` snapshot will be used to create
 
 * `--ci`: If this is set, all unrecognized arguments will be treated as a new command. This command will be run once the test chain is up, and Ganache will exit afterward.
 
+* `-o, --output`: Works only with `-d`. Will export deployed contracts to specified format. Available formats are: `json|env|yaml|template`
+
+* `--template`: Output configuration template. See (deployment output section)[#deploying-changes-to-contract-code].
+
 ## Deploying changes to contract code
 
-When the deploy script is run, it copies ABI files and an `addresses.json` file containing the addresses of deployed contracts to `out/`. These can be copied into the source code of applications working with the test chain as you see fit.
+When the deploy script is run, it copies ABI files and output addresses of deployment contracts to file in `out/` directory. These can be copied into the source code of applications working with the test chain as you see fit.
+
+This output file can be in different format depending on `-o, --output` option.
+
+ * `-o json` - creates `out/addresses.json` file in JSON format
+ * `-o env` - creates `out/addresses` file with bash script that setting variables to ENV params.
```

Comment:
grammer nitpick:

"creates `out/addresses` bash script that exports the addresses as environment variables"